lib/src/base-styles/_nx-toast.scss

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../base-styles/nx-alert';
+@use '../scss-shared/nx-container-helpers';
+
+.nx-toast__wrapper {
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  @include nx-container-helpers.container-vertical;
+  margin-top: var(--nx-spacing-2x);
+  position: fixed;
+}
+
+.nx-toast {
+  @extend .nx-alert;
+  margin: var(--nx-spacing-2x) 0 0 0;
+  opacity: 0.5;
+  position: relative;
+  width: 500px;
+}
+
+.nx-toast--success {
+  @extend .nx-alert--success;
+}
+
+.nx-toast--warning {
+  @extend .nx-alert--warning;
+}
+
+.nx-toast--error {
+  @extend .nx-alert--error;
+}
+
+.nx-toast--info {
+  @extend .nx-alert--info;
+}
+
+.animate {
+  transition: all 0.3s ease-in-out;
+}
+
+.animate.slide-in {
+  opacity: 1;
+  top: 0;
+}
+
+.animate.slide-out {
+  opacity: 0.5;
```

Comment:
Why not 0?

Reply:
> Why not 0?

I removed `.slide-out` during the commit below. `.slide-out`'s properties were moved to `.nx-toast`, which now has opacity of 0.

[8ffc769](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/8ffc7693bf6a8fb6b79b243f92e6c674babd8dc5)